fuel-core/src/config/chain_config.rs

```diff
@@ -121,6 +120,16 @@ pub struct StateConfig {
     pub height: Option<BlockHeight>,
 }
 
+impl StateConfig {
+    pub fn generate_state_config(db: Database) -> Self {
+        StateConfig {
+            coins: db.get_coin_config().unwrap(),
```

Comment:
are these things always assumed to work or should we be more careful with `unwraps` here ?

Reply:
Moving this to `unwrap_or_default`, these should always work and properly return a `None` value if nothing exists, but moving to `unwrap_or_default` so an error in one of them doesn't ruin the generation of the entire config